app/models/legislation.rb

```diff
@@ -12,7 +12,8 @@ def self.all
         short: _('FOI'),
         full: _('Freedom of Information'),
         with_a: _('A Freedom of Information request'),
-        act: _('Freedom of Information Act')
+        act: _('Freedom of Information Act'),
+        refusals: ['s 12', 's 14', 's 40', 's 21', 's 31']
```

Comment:
This will be done in the theme

Reply:
Better to remove it from this commit and create a PR for WDTK theme now that https://github.com/mysociety/alaveteli/pull/5931 is merged?